ingress.tf

```diff
@@ -0,0 +1,24 @@
+data "template_file" "tls_secret" {
```

Comment:
why `template_file` instead of `jsone_template` (here and a few other places)?

Reply:
Hahaha, I use it to base64encode things since we don't have that in json-e yet.